jacob-george/starter-code/package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "mvc-blog",
+  "version": "1.0.0",
+  "description": "As we now complete our MVC conversation, we can refactor our app to use good MVC separation of concerns.",
+  "main": "server.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "Code Fellows",
+  "license": "ISC",
+  "dependencies": {
+    "body-parser": "^1.15.2",
+    "express": "^4.13.3",
+    "pg": "^6.1.2"
```

Comment:
When you `npm install express-request-proxy` make sure to add `--save` at the end so it saves onto your package.json.